desktop/package.json

```diff
@@ -86,5 +88,8 @@
     "webpack-dev-server": "1.14.1",
     "webpack-hot-middleware": "2.10.0",
     "webpack-target-electron-renderer": "0.4.0"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.0.11"
```

Comment:
Fsevents isn't a direct dependency and I think in the sub dependency it's
optional already

On Friday, May 20, 2016, Max Goodman <notifications@github.com> wrote:

> In desktop/package.json
> <https://github.com/keybase/client/pull/2822#discussion_r64082269>:
>
> > @@ -86,5 +88,8 @@
> >      "webpack-dev-server": "1.14.1",
> >      "webpack-hot-middleware": "2.10.0",
> >      "webpack-target-electron-renderer": "0.4.0"
> > +  },
> > +  "optionalDependencies": {
> > +    "fsevents": "1.0.11"
>
> This became necessary because the CI scripts now do a full npm install.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2822/files/0a59e9831a981a15364ceeb793ff1341cbffb879#r64082269>
>